spec/support/helpers.rb

```diff
@@ -4,6 +4,7 @@ module Helpers
   def with_isolated_config
     original_config = RSpec.configuration
     RSpec.configuration = RSpec::Core::Configuration.new
+    RSpec.configuration.world = RSpec.world
```

Comment:
Why it was needed?

Reply:
Otherwise, the two worlds are out of sync inside `with_isolated_config`:
```ruby
    with_isolated_config do
      RSpec::Core::ExampleGroup.describe('a view') { it { } }
      RSpec.configuration.world.example_groups # => []
    end
```
without `with_isolated_config`:

```ruby
      RSpec::Core::ExampleGroup.describe('a view') { it { } }
      RSpec.configuration.world.example_groups # => [<Example ...>]
```

I got rid of `with_isolated_config` in this example, so this change is to make `with_isolated_config` more future-proof.

Reply:
If this doesn't fix a failing spec I'm going to say we should leave this as it was on this branch at least.

Reply:
Removed. Let's see if it affects the build. At least it this change doesn't seem to affect `view_example_group_spec.rb` on 2.2.10/5.2.4.3.